DESCRIPTION

```diff
@@ -35,6 +36,8 @@ Suggests:
     r2rtf (>= 0.3.2),
     rmarkdown (>= 2.19),
     stringi (>= 1.6),
+    stringr (>= 1.5.0),
```

Comment:
We did a lot of work to avoid importing {stringr} and having only {stringi} in suggests so to have a lighter dep tree as this package is the basis of much of what we do. We have already implemented some ways to read in any `xx` input, we just lacked the time to add it to {formatters}